operator/data/examples/multicluster/values-istio-multicluster-primary.yaml

```diff
@@ -2,31 +2,33 @@ apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
   values:
-    gateways:
-      istio-ingressgateway:
-        env:
-          ISTIO_META_NETWORK: "network1"
+    # selfSigned is required if Citadel is enabled, that is when values.global.istiod.enabled is false.
```

Comment:
This does not currently exist anymore - citadel is not around in 1.6

Reply:
I still see it in default.yaml for default profile.https://github.com/istio/istio/blob/master/manifests/profiles/default.yaml#L104